docs/dictionary/property/alwaysBuffer.lcdoc

```diff
@@ -23,24 +23,45 @@ set the alwaysBuffer of image "Smile" to false
 Example:
 set the alwaysBuffer of the templatePlayer to true
 
-Value (bool): The <alwaysBuffer> of an <object(glossary)> is true or false. By <default>, the <alwaysBuffer> <property> of newly created <image|images>, <player|players>, and <stacks> is set to false.
+Value (bool): The <alwaysBuffer> of an <object> is true or false. By <default>, 
```

Comment:
Could you reinstate the `<object(glossary)>` please? It seems to break things in the dictionary when there 's an `<object>` on it's own because it's a valid html tag!